neqo-common/Cargo.toml

```diff
@@ -16,27 +16,25 @@ readme.workspace = true
 workspace = true
 
 [dependencies]
-# Checked against https://searchfox.org/mozilla-central/source/Cargo.lock 2024-11-11
 enum-map = { workspace = true }
 env_logger = { version = "0.10", default-features = false }
-hex = { version = "0.4", default-features = false, features = ["alloc"], optional = true }
+hex = { workspace = true, optional = true }
 log = { workspace = true }
 qlog = { workspace = true }
 strum = { workspace = true }
 
 [target."cfg(windows)".dependencies]
-# Checked against https://searchfox.org/mozilla-central/source/Cargo.lock 2024-11-11
 windows = { version = "0.58", default-features = false, features = ["Win32_Media"] }
 
 [dev-dependencies]
 criterion = { version = "0.5", default-features = false }
 neqo-crypto = { path = "../neqo-crypto" }
 test-fixture = { path = "../test-fixture" }
-regex = { workspace = true }
+regex = { workspace = true, features = ["perf"] }
```

Comment:
Do I understand correctly that we didn't enable `perf` before? I would prefer enabling it in a separate pull request to see whether it has an actual performance impact.

For others, feature details:

https://docs.rs/regex/latest/regex/#performance-features

Reply:
I'll add it to #2477.